db/migrate/20160913182424_create_offsprings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOffsprings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :offsprings do |t|
+      t.string :first_name
+      t.string :last_name
```

Comment:
Offspring should belong to one User, you will need to make another migration afterwards